networking/nw-create-custom-ingress-controller.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-operator.adoc
+
+:_content-type: PROCEDURE
```

Comment:
I'm not sure why this module is being placed in the Networking directory. It looks like it duplicates the procedure placed in the modules directory with a few differences.